native/KotlinIOS/KotlinIOS/customTableViewCell.swift

```diff
@@ -1,15 +1,17 @@
-//
-//  customTableViewCell.swift
-//  KotlinIOS
-//
-//  Created by Gabriel Swallow on 28/06/2022.
-//  Copyright © 2022 Evgeny Petrenko. All rights reserved.
-//
-
 import UIKit
-
+import SharedCode
 class customTableViewCell: UITableViewCell {
 
+    @IBOutlet var departureStation: UILabel!
+    @IBOutlet var departureTime: UILabel!
+
+    @IBOutlet var arrivalStation: UILabel!
+    @IBOutlet var arrivalTime: UILabel!
+    
+    @IBOutlet var carrier: UILabel!
```

Comment:
Nit: Could we call this operator?

Reply:
operator is a keyword in Swift, so we've renamed it to trainOperator